### ISSUE TITLE
classification example - custom scoring metric

### ISSUE THREAD

**Issue by jonas-sprindys** [stanfordmlgroup/ngboost#38]
In the classification example (https://github.com/stanfordmlgroup/ngboost/blob/master/examples/empirical/clf_sklearn.py) , I tried to provide custom scoring metric:

grid_search = GridSearchCV(ngb, param_grid=param_grid, scoring = 'roc_auc', cv=5)

However, the following error is raised: 
'NGBClassifier' object has no attribute 'predict_proba'

**Comment by tonyduan** [comment]
Closing this issue since it's been addressed by PR #44.

_However_, I would advise that in general when using NGBoost, hyper-parameters are chosen (via cross validation or the like) to maximize the _explicitly specified score_ i.e. MLE or CRPS, instead of another objective such as ROC. Note that this recommendation is the behavior by default (see examples/sklearn_cv.py), due to the implementation of the `score` function in the NGBoost class. 

All that said, we've added support for `predict_proba` in case you really need to cross validate for ROC in your workflow.
